packages/ui/src/elements/SaveDraftButton/index.tsx

```diff
@@ -91,7 +93,7 @@ export const SaveDraftButton: React.FC = () => {
       buttonId="action-save-draft"
       buttonStyle="secondary"
       className={baseClass}
-      disabled={forceDisable}
+      disabled={forceDisable || uploadStatus === 'uploading'}
```

Comment:
It looks like you missed line 80, which checks for forceDisable but not uploadStatus. I think you can still save with the hotkey.

It would be better to set forceDisable and pass it down through props. That also would make it so that custom buttons don't have to also have this extra new logic added as they should already respect `forceDisable` props.